### PR TITLE
storage: default booking size

### PR DESCRIPTION
**Pull request by lnielsen**

### CODE REVIEW

**Comment by lnielsen** [comment]
https://travis-ci.org/lnielsen/invenio-xrootd/builds/157871597

File: invenio_xrootd/storage.py [hunk 1]
```diff
@@ -121,20 +121,23 @@ class EOSFileStorage(XRootDFileStorage):
     clients **must** provide the file size upfront.
     """
 
-    def __init__(self, *args, **kwargs):
+    def __init__(self, fileurl, default_booking_size=None, **kwargs):
         """Initialize storage."""
         self.bookingsize = None
-        super(EOSFileStorage, self).__init__(*args, **kwargs)
+        self.default_booking_size = default_booking_size or \
+            (current_app.config.get('MAX_CONTENT_LENGTH') if current_app
```
Comment (jirikuncar) [inline invenio_xrootd/storage.py:128]: `if current_app` -> `if has_app_context` (http://flask.pocoo.org/docs/0.11/api/#flask.has_app_context)

Comment (lnielsen) [reply invenio_xrootd/storage.py:128]: > Works like has_request_context() but for the application context. You can also just do a **boolean check on the current_app** object instead.

Comment (jirikuncar) [reply invenio_xrootd/storage.py:128]: Sure you can. It's just that `has_app_context` is more readable and IMHO expresses the intent better.

**Pull request merged**